docs/writing-tests/idlharness.md

```diff
@@ -129,6 +104,18 @@ and outside callers should not use it.
   `document.head` would only be tested for `HTMLHeadElement`, and so on for
   further objects.
 
-### `test()`
-  Run all tests.  This should be called after you've called all other
-  methods to add IDLs and objects.
+### `add_idls(idl_string)`
```

Comment:
Nit; it may be worth noting that this is discouraged, and in general IDL should be in an `/interfaces/` file and loaded via the `srcs` parameter?